src/patternfly/components/DataList/docs/data-list-check-action.md

```diff
@@ -0,0 +1,17 @@
+### Data List Checkbox/Action Button Notes
+
+<mark>**All simple data list accessibility and usage requirements apply.**</mark>
+
+### Accessibility
+| Attribute | Applied To | Outcome |
+| -- | -- | -- |
+| `aria-label="[descriptive text]"`                                       | `.pf-c-data-list__action` > `.pf-c-button`                                                    | Provides an accessible label buttons. **Required** |
+| `aria-labelledby="{title_cell_id} {data_list_interactive_element_id}"`  | `.pf-c-data-list__check` > `.pf-c-check__input`, `.pf-c-data-list__action` > `.pf-c-button`   | Establishes relationship between interactive element and aria-label. **Required** |
```

Comment:
```suggestion
| `aria-labelledby="{title_cell_id}"`  | `.pf-c-data-list__check` > `.pf-c-check__input`   | Creates an accessible label for the checkbox based on the title cell. **Required** |
| `aria-labelledby="{title_cell_id} {data_list_action_id}"`  | `.pf-c-data-list__action` > `.pf-c-button`   | Creates an accessible label for the action button using the title cell and button label **Required** |
```

Reply:
I like consolidating rows whenever possible, but in this case, only the action button uses multiple id values in `aria-labelledby`.